src/sinks/vector/v1.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{
-    config::{DataType, GenerateConfig, SinkConfig, SinkContext, SinkDescription},
+    config::{DataType, GenerateConfig, SinkConfig, SinkContext},
```

Comment:
Original implementation moved, and some type names/imports updated to unify the v1/v2 terminology.